app/templates/compare_revisions.html

```diff
@@ -63,27 +63,19 @@
     <div class="grid-row">
       <div class="column-two-thirds page-section">
         <p>
-          {% if all_update_events is none %}
-            Multiple
+          {% if latest_update_events %}
+            {% if number_of_users_who_made_edits > 1 %}
+              The last user to edit this page was
+            {% endif %}
+              {{ latest_update_events.0.user }}
+            {% if number_of_users_who_made_edits == 1 %}
+              made {{ (all_update_events or (oldest_update_events+latest_update_events))|length }}
+              {{ pluralize((all_update_events or (oldest_update_events+latest_update_events))|length, "edit", "edits") }}
+            {% endif %}
+              on {{ latest_update_events[0].createdAt|dateformat -}}
           {% else %}
             {{ all_update_events|length }}
-          {% endif %}
-          unapproved {{ pluralize((all_update_events or (oldest_update_events+latest_update_events))|length, "edit", "edits") }}
-          {%- if latest_update_events %}
-            {% if all_update_events is not none %}
-              by
-              {% if n_editing_users_min == 1 %}
-                {{ latest_update_events.0.user }}
-              {% else %}
-                {{ n_editing_users_min }} users
-              {% endif %}
-            {% endif %}
-            {% if latest_update_events[0].createdAt|dateformat == oldest_update_events[-1].createdAt|dateformat %}
-              on {{ latest_update_events[0].createdAt|dateformat }}
-            {%- else %}
-              between {{ oldest_update_events[-1].createdAt|dateformat }}
-              and {{ latest_update_events[0].createdAt|dateformat }}
-            {%- endif -%}
+            unapproved {{ pluralize((all_update_events or (oldest_update_events+latest_update_events))|length, "edit", "edits") -}}
```

Comment:
I don't know if I really see the point in this change. I _did actually_ think this all this wording out quite hard based on the information we are actually able to _reliably_ show people, and it was only a few months ago. Is this churn really necessary?

Reply:
This change has been made following user research done with people using admin page. They wanted to see who made last edit if more than one person made an edit, to be able to contact that person.  Earlier this information was not shown. After the need / pain point was recognised, it was run by our team, including product manager and content designer, and this ticket is the result of this process.
If you would like to question it, I think the best person might be Tobi.

Reply:
It would have been really easy to prepend the most recent editor's email to the sentence in the case when a user wasn't shown and make minimal changes. There are a million things that *really need doing* on this page and instead of building the road forward we're deciding to tear up the bit that I've just tarmac'ed.

Reply:
I agree with Rob - it would really be a shame to lose the nuance and additional information here.  I know the ticket says to use a blanket "More than one user has edited this service.' but that will only work until the day comes when admin need more information.

The ticket is about showing who the last user to edit the service was.  Like Rob says, the easy win here is just to add "The last user to edit this service was X" to the existing information where necessary.

Reply:
@CrystalPea - to follow up on your question - the current version of the page gives admin users the most complete and accurate summary of the changes that have been made to the service that we can, given the data available to the view.  It seems to me that admin users should be interested in a complete and accurate picture of what has gone on.

So I'd rather see only the addition of the extra info here (i.e. who made the latest change, as the headline of the ticket says), rather than losing this detail.

Reply:
Ok we can talk about this tomorrow on standup with Tobi and Constance